test/org/sagebionetworks/bridge/services/AuthenticationServiceTest.java

```diff
@@ -255,6 +256,9 @@ public void canResendEmailVerification() throws Exception {
 
     @Test
     public void createResearcherAndSignInWithoutConsentError() {
+        BridgeUtils.setRequestContext(new RequestContext.Builder()
+                .withCallerRoles(ImmutableSet.of(Roles.ADMIN)).build());
+        
         testUser = helper.getBuilder(AuthenticationServiceTest.class)
                 .withConsent(false).withSignIn(false).withRoles(Roles.RESEARCHER).build();
```

Comment:
The request context says admin, but the user says researcher.

Reply:
This is correct. The AuthenticationService will now verify the caller is an ADMIN, so RequestContext must be set. I've added a comment to the two tests that set this.